### PR TITLE
Add module website_sale_suggest_create_account.

### PR DESCRIPTION
**Pull request by yajo**
This module was written to extend the functionality of e-Commerce to suggest the buyer to create an account, but without forcing him to do it.

CC @rafaelbn.

### CODE REVIEW

File: website_sale_suggest_create_account/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+However, the *Checkout without sign in* button is not highlighted, to
+implicitly encourage users to create the account.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* *Settings > Configuration > General Settings* and enable *Allow external
```
Comment (oscarolar) [inline website_sale_suggest_create_account/README.rst:23]: Can you set this as a default value so no configuration is needed? (.yml or .xml files)

Comment (yajo) [inline website_sale_suggest_create_account/README.rst:23]: Allowing anybody to sign into your Odoo implies some security considerations, so I think this step should be made by hand. Anyway this parameter has no XMLID to inherit from, so does not seem possible to do it with data files.

**Comment by carlosdauden** [comment]
:+1:

**Comment by rafaelbn** [comment]
Tested in runbot :+1:

**Comment by dgzurita** [comment]
Tested in runbot :+1:

**Pull request merged**